test/e2e/common/node_lease.go

```diff
@@ -84,6 +84,30 @@ var _ = framework.KubeDescribe("NodeLease", func() {
 				time.Duration(*lease.Spec.LeaseDurationSeconds/4)*time.Second)
 		})
 
+		ginkgo.It("should have OwnerReferences set", func() {
```

Comment:
Should this be a conformance test?

Reply:
We don't create conformance test from scratch.
At some point it probably should be, but we need to create it as is and after proving its stability promote it to conformance.